app/api/privy/integrations/discord/interactions.rb

```diff
@@ -0,0 +1,20 @@
+module Privy
+  module Integrations
+    module Discord
+      class Interactions < Grape::API
+        desc 'Receive interactions from Bot'
+
+        post '/interactions' do
+          raw_body = request.body.read
+          interaction_request = ::Discord::Interactions::Request.new(params:, headers:, raw_body:)
+          @resolver = ::Discord::Interactions::Resolvers::Resolver.create_resolver(request: interaction_request)
```

Comment:
just curious, why do you use `::Discord::Interactions::Request` instead of `Discord::Interactions::Request`? `::` at the beginning should be used only when there is an ambiguous definition in the namespace and you need to tell ruby to find the class in the global scope instead of this module's scope. But it should not be used by default, does it break if you don't use it?

Reply:
look, for example:

<img width="808" alt="image" src="https://github.com/user-attachments/assets/1cc295be-b1c1-4518-ad13-c0439d69a566">